pages/docs/installation/installation-special-systems.md

```diff
@@ -16,6 +16,52 @@ The instructions may still be valuable for unlisted systems.
 
 ## Active systems
 
+### HAWK (HPE Apollo/AMD, Stuttgart)
+
+#### Building
+
+The following steps explain how to install preCICE on HAWK with PETSc and MPI (using the system standard MPE):
```

Comment:
What is "MPE"? Does this refer to the MPI version?

Reply:
Yes, it refers to the HPE MPI implementation: https://kb.hlrs.de/platforms/index.php/MPI(Hawk)

Reply:
Then let's add this link to the page! :smile: